tools/saucelabs-daemon/background-service/saucelabs-daemon.ts

```diff
@@ -195,20 +194,24 @@ export class SaucelabsDaemon {
   private async _connect() {
     await openSauceConnectTunnel(
         (this._userCapabilities as any).tunnelIdentifier, this._sauceConnect);
-    await this._launchBrowsers();
   }
 
   /**
    * @internal
-   * Launches all browsers. If there are pending tests waiting for a particular browser to launch
-   * before they can start, those tests are started once the browser is launched.
+   * Launches a set of browsers and increments the count of parallel browser started. If there are
+   * pending tests waiting for a particular browser to launch before they can start, those tests are
+   * started once the browser is launched.
    **/
-  private async _launchBrowsers() {
-    console.debug('Launching browsers...');
+  private async launchBrowsers() {
```

Comment:
just to make it clear that this is not launching _all_ possible browsers

```suggestion
  private async launchBrowserSet() {
```

Reply:
👍